addons/website/static/tests/builder/save.test.js

```diff
@@ -262,3 +269,34 @@ function setupSaveAndReloadIframe() {
     });
     return resultSave;
 }
+
+// Mock the action that is invoked on switchTheme
+defineModels([
+    class extends models.Model {
+        _name = "ir.module.module";
+    },
+]);
+defineActions([
+    {
+        id: 99,
+        xml_id: "website.theme_install_kanban_action",
+        name: "Pick a Theme",
+        res_model: "ir.module.module",
+        target: "fullscreen",
+        type: "ir.actions.act_window",
+        views: [[false, "form"]],
+    },
+]);
```

Comment:
You can use a client action to make it easier to read and you don t need defineModels with this solution. You can also add a expect.step to check if the action is call